### DISCUSSION TITLE
Moving RavenDB to a new Server with Same Url

### DISCUSSION THREAD

**Discussion by tzarger** [ravendb/ravendb#15053] [Q&A] [answered]
Hello, we are moving away from AWS and I am setting up another server at another provider. What is the best was to simply move the server, move DNS and update the DNS to point to RavenDB server as before? I know there is Smuggler, but the issue I would face is I want the old AWS instance URL to be the same as the new Server Instance. 

Would it be set up another node in the cluster? then remove the original node? If I do that, wouldn't need to be a different URL for the second node and how would that affect the document IDs?

Any help would be appreciated. Thanks!

**Comment by ayende** [comment, marked as answer]
Let's start from the basic, easier would be to:

* Shut down the server
* Move the data to the new server
* Update DNS
* Start the new server

The only thing you actually need to do is update the DNS

The above will require downtime.

Without downtime, you need to:

* Setup a new server (with new DNS)
* Add that server to the existing cluster
* Add that server to the databases you want to migrate 
* Let it replicate and index the data
* Shutdown the old server

Move the DNS / have the new one, as you wish.


Regardless, document ids are not impacted by the server URL
